internal/distro/rhel8/images.go

```diff
@@ -396,11 +397,14 @@ func edgeRawImage(workload workload.Workload,
 
 	// "rw" kernel option is required when /sysroot is mounted read-only to
 	// keep stateful parts of the filesystem writeable (/var/ and /etc)
-	img.KernelOptionsAppend = []string{"modprobe.blacklist=vc4", "rw"}
+	img.KernelOptionsAppend = []string{"modprobe.blacklist=vc4"}
 	// TODO: move to image config
 	img.Keyboard = "us"
 	img.Locale = "C.UTF-8"
-	img.SysrootReadOnly = true
+	if !common.VersionLessThan(t.arch.distro.osVersion, "8.8") || t.arch.distro.osVersion == "8-stream" {
+		img.SysrootReadOnly = true
+		img.KernelOptionsAppend = append(img.KernelOptionsAppend, "rw")
+	}
```

Comment:
I'm reluctant at this point to ship this for 8.8 since we're not providing a migration mechanism for users to enable RO - for 9.2 for instance, users coming from <9.2 (including 8.x) will get RO, but if you upgrade from 8.7 (RW) to 8.8 you still get (RW) as 8.8 doesn't have the migration code but 8.8 gets RO only on newer installation - ok, I'm gonna drop all this